po/pl.po

```diff
@@ -833,19 +866,19 @@ msgstr "Wystąpił nieznany błąd."
 
 #: ../src/main.cc:138
 msgid "Print information on the indicated element(s) to standard output"
-msgstr ""
+msgstr "Wydrukuj informacje wskazanym/ch pierwiastku/ach na wyjściu standardowym"
```

Comment:
Looks like a word is missing. I think it should be "Wydrukuj informacje **o** wskazanym/ch pierwiastku/ach na wyjściu standardowym"